en/django_forms/README.md

```diff
@@ -44,32 +44,38 @@ So once again we will create a link to the page, a URL, a view and a template.
 
 ## Link to a page with the form
 
-It's time to open `blog/templates/blog/base.html` in the code editor. In the `div` named `page-header`, we will add a link:
+It's time to open `blog/templates/blog/base.html` in the code editor. In the `div` named `container` inside `page-header` `div` tag, we will add a link:
 
 {% filename %}blog/templates/blog/base.html{% endfilename %}
 ```html
-<a href="{% url 'post_new' %}" class="top-menu"><span class="glyphicon glyphicon-plus"></span></a>
+<a href="{% url 'post_new' %}" class="top-menu">
+    <svg width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-file-earmark-plus" fill="currentColor" xmlns="http://www.w3.org/2000/svg"><path d="M4 0h5.5v1H4a1 1 0 0 0-1 1v12a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1V4.5h1V14a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2z"/><path d="M9.5 3V0L14 4.5h-3A1.5 1.5 0 0 1 9.5 3z"/><path fill-rule="evenodd" d="M8 6.5a.5.5 0 0 1 .5.5v1.5H10a.5.5 0 0 1 0 1H8.5V11a.5.5 0 0 1-1 0V9.5H6a.5.5 0 0 1 0-1h1.5V7a.5.5 0 0 1 .5-.5z"/></svg>
```

Comment:
From a developer perspective I understand this, but if I was a student this would confuse me completely. I think this deserves a bit more explanation.

Reply:
I have thought about that. As it conserned to bootstrap4, they dont provide a icon package as css. This is where complexity begun. Next best option is font awesome. It is very similar to what it already have(glyphicon). But they have change the way of providing CND link publicly. I can still get the link from other CDN provides.

Then I thought of pushing the default icons of Bootstrap as SVG initially and change it to font awesome if required with an aditional commit to this PR.

If we are doing that, we need to fix up the best possible version of Font Awesome to be use.

One more thing is about using `integrity` and `crossorigin` attributes in `<link>` tags. @ekohl As you said, this decreases the readablility of code and increase complexity for users.

If we were to keep using the Bootstrap4 icons instead of Font awesome, I really feel like saving these svg files as assets and using it from `{% static 'img/icon-new-post.svg' %}` with `<img>` tag or `{% include './icons/new-psot.svg' %}`. The descriptive information regarding the same have to be added as well.

Reply:
@ekohl Please suggest how to proceed?

Reply:
> One more thing is about using `integrity` and `crossorigin` attributes in `<link>` tags. @ekohl As you said, this decreases the readablility of code and increase complexity for users.

I don't mind these too much. On https://getbootstrap.com/ they use the same thing so it'd be the same thing if they were to follow that.

> If we were to keep using the Bootstrap4 icons instead of Font awesome, I really feel like saving these svg files as assets and using it from `{% static 'img/icon-new-post.svg' %}` with `<img>` tag or `{% include './icons/new-psot.svg' %}`. The descriptive information regarding the same have to be added as well.

This sounds like a nice solution.